.golangci.yml

```diff
@@ -121,7 +121,7 @@ linters-settings:
       - name: unhandled-error
       - name: unnecessary-stmt
       - name: unreachable-code
-      - name: unused-parameter
+      # - name: unused-parameter
```

Comment:
Disabled `unparam` and `unused-parameter` are covered here: https://github.com/influxdata/telegraf/pull/8984 :)